hyperspy/signal.py

```diff
@@ -1205,34 +1208,152 @@ def export_decomposition_results(self, comp_ids=None,
+                             no_nans=no_nans,
+                             same_window=same_window,
+                             per_row=per_row,
+                             save_figures_format=save_figures_format)
+        self._export_loadings(loadings,
+                              comp_ids=comp_ids, folder=folder,
+                              calibrate=calibrate,
+                              multiple_files=multiple_files,
+                              loading_prefix=loading_prefix,
+                              loading_format=loading_format,
+                              comp_label=comp_label,
+                              cmap=cmap,
+                              save_figures=save_figures,
+                              same_window=same_window,
+                              no_nans=no_nans,
+                              per_row=per_row)
+
+    def export_cluster_results(self,
+                               cluster_ids=None,
+                               folder=None,
+                               calibrate=True,
+                               center_prefix='cluster_center',
+                               center_format="hspy",
+                               membership_prefix='cluster_label',
+                               membership_format="hspy",
+                               comp_label=None,
+                               cmap=plt.cm.gray,
+                               same_window=False,
+                               multiple_files=True,
+                               no_nans=True,
+                               per_row=3,
+                               save_figures=False,
+                               save_figures_format='png'):
+        """Export results from a cluster analysis to any of the supported
+        formats.
+
+        Parameters
+        ----------
+        cluster_ids : None, int, or list of ints
+            if None, returns all clusters/centers.
+            if int, returns clusters/centers with ids from 0 to
+            given int.
+            if list of ints, returnsclusters/centers with ids in
+            given list.
+        folder : str or None
+            The path to the folder where the file will be saved.
+            If `None` the
+            current folder is used by default.
+        center_prefix : string
+            The prefix that any exported filenames for
+            cluster centers
+            begin with
+        center_format : string
+            The extension of the format that you wish to save to. Default is
+            "hspy". See `loading format` for more details.
+        label_prefix : string
+            The prefix that any exported filenames for
+            cluster labels
+            begin with
+        label_format : string
+            The extension of the format that you wish to save to. default
+            is "hspy". The format determines the kind of output.
+            - For image formats (tif, png, jpg, etc.), plots are
```

Comment:
You need a blank newline before the list, otherwise Sphinx doesn't format the bulleted list correclty and text gets lost.
So:
```
        label_format : string
            The extension of the format that you wish to save to. default
            is "hspy". The format determines the kind of output.
            - For image formats (tif, png, jpg, etc.), plots are
```
becomes
```
        label_format : string
            The extension of the format that you wish to save to. default
            is "hspy". The format determines the kind of output.

            - For image formats (tif, png, jpg, etc.), plots are
```

Reply:
Fixed ! , Thanks Thomas